src/Archives/HuffLZ.cpp

```diff
@@ -4,10 +4,10 @@
 namespace Archives
 {
 	// Constructs the object around an existing bit stream
-	HuffLZ::HuffLZ(BitStreamReader *bitStream) :
+	HuffLZ::HuffLZ(BitStreamReader& bitStream) :
 		m_BitStreamReader(bitStream),
-		m_ConstructedBitStreamReader(0), // Don't need to delete stream in destructor
-		m_AdaptiveHuffmanTree(new AdaptiveHuffmanTree(314)), 
+		m_ConstructedBitStreamReader(), // Don't need to delete stream in destructor
```

Comment:
I find the empty parenthesis a bit odd here. I'm wondering if we should be more explicit about this. Passing a `nullptr` would be appropriate for the initialization value.